Dockerfile

```
@@ -19,12 +19,12 @@ RUN apt-get update && apt-get install -y \
 # ZSH Installation
 RUN apt-get update && apt-get install zsh -y
 RUN sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
-RUN echo "source /opt/ros/eloquent/setup.zsh" >> ~/.zshrc
+RUN echo "source /opt/ros/foxy/setup.zsh" >> ~/.zshrc
 
 # Install Gazebo
 RUN sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
 RUN wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
-RUN apt-get update && apt-get install -y gazebo9 libgazebo9-dev ros-eloquent-gazebo-ros-pkgs
+RUN apt-get update && apt-get install -y gazebo11 libgazebo11-dev ros-foxy-gazebo-ros-pkgs
```

```suggestion
RUN apt-get update && apt-get install -y gazebo11 libgazebo11-dev
```

I would suggest to let `ros-foxy-gazebo-ros-pkgs` be handled by `rosdep`